service/routes.py

```diff
@@ -396,7 +405,10 @@ def post(self):
                 "routes.py, InventoryCollection::post, an error occurred: %s",
                 error,
             )
-            return "", status.HTTP_409_CONFLICT
+            abort(
+                    status.HTTP_409_CONFLICT,
+                    f"Inventory with id {inventory.product_id} and condition {api.payload['condition']} already exists",
+                )
```

Comment:
Earlier there was no message indication on the UI incase of `IntegrityError` from backend. Added these `abort` statements to indicate the correct failure message on UI.